static/js/app.ts

```diff
@@ -42,6 +44,12 @@ export let theGlobalEditor: AceAjax.Editor;
     }
   }
 
+  function initializedExampleCounter() {
+    if (window.State.examples_left <= 3) {
```

Comment:
Shall we place the 3 somewhere so that it is easy to customize? Maybe teachers should be able to set it for their classes? And maybe it should also be dependent on the level (unlimited for 1, 10 for 2, 3 for 3 etc.)?

Reply:
Agree! Not sure what would be the best place to store this value, will take a look!

Reply:
I just noticed this function is redundant as we don't give inline feedback to the user anymore. It is removed. To keep track of the amount of examples_left we use a `window.State` value called `examples_left`. Which is set to 3 if the user is logged in, otherwise it is set to max int

Reply:
ok!